src/Model/MethodGo.cs

```diff
@@ -125,9 +125,14 @@ public string MethodParametersSignature(bool includePkgName = false)
 
         private string ParameterTypeSig(IModelType type, bool includePkgName)
         {
-            if (includePkgName && type is CompositeTypeGo)
+            if (includePkgName)
             {
-                return $"{CodeModel.Namespace}.{type.Name}";
+                if (type.IsUserDefinedType())
+                    return $"{CodeModel.Namespace}.{type.Name}";
+                else if (type is SequenceTypeGo stg)

```

Comment:
why do we need the package prefix on these ? wont all the types be in the same package ?

Reply:
no the generated interfaces are in a sub-package.

Reply:
oh ok, for the interfaces cuz they are in different packages.

Reply:
ah vs code comments dont sink very well